### PR TITLE
183 dashboard fuzzy search

### LINKED ISSUE #183: Dashboard Fuzzy Search

### PR DESCRIPTION
**Pull request by mvallido**
closes #183

### CODE REVIEW

File: src/app/Http/Controllers/DashboardController.php [hunk 1]
```diff
@@ -29,9 +30,13 @@ public function index()
...
-        $query_numRooms = $request -> query('query_numRooms');
-        $addressObj = Accommodation::where('address', 'LIKE', '%'.$query_address.'%')->where('numRooms', 'LIKE', '%'.$query_numRooms.'%')->paginate(10);
-        return $addressObj;
+        $query = $request -> query('query');
+
+        $addressObj = Accommodation::search($query)->get();
+        $groupObj = Group::search($query)->get();
+        $bookingObj = Booking::search($query)->get();
+
+        $arr = array($addressObj,$groupObj,$bookingObj);
```
Comment (GMcLachlan45) [inline src/app/Http/Controllers/DashboardController.php:39]: So all of these objects are being searched, queried, and sorted, but they arent being sorted amongst themselves.

Is this something that will be addressed in the future?

**Review by GMcLachlan45 — COMMENTED** [review]
Looks great, if I can get some deliberation on that one item, then you'll be good to go.

File: src/app/Http/Controllers/DashboardController.php (hunk 1 above)
Comment (mvallido) [reply src/app/Http/Controllers/DashboardController.php:39]: Yes, I'll address this after I complete my next task. I'll create an issue for it.

Comment (GMcLachlan45) [reply src/app/Http/Controllers/DashboardController.php:39]: Ok sounds goo

**Review by GMcLachlan45 — APPROVED** [review]
Feel free to merge then.

**Pull request merged**